package/boot/uboot-tools/uboot-envtools/files/mediatek_filogic

```diff
@@ -92,6 +92,7 @@ zbtlink,zbt-z8103ax)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x20000" "0x20000"
 	;;
 dlink,aquila-pro-ai-m30-a1|\
+dlink,aquila-pro-ai-m30-a1-ubootmod|\
```

Comment:
Please use a more accurate suffix instead of `ubootmod`. This new partition layout does not require replacing u-boot.